common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -190,7 +190,8 @@
                 DiscussionUtil.clearFormErrors(this.$('.post-errors'));
                 this.$('.wmd-preview p').html('');
                 if (this.isTabMode()) {
-                    this.topicView.setTopic(this.$('button.topic-title').first());
+                    var $general = this.$('.post-topic option:contains(General)');
+                    this.topicView.setTopic($general ? $general : this.$('button.topic-title').first());
```

Comment:
`$general || this.$('button.topic-title').first()` is a shorter way to express this, I think